opm/input/eclipse/Schedule/Well/WellFractureSeeds.hpp

```diff
@@ -49,6 +50,169 @@ class WellFractureSeeds
     /// Type alias for the normal vector at a single seed point.
     using NormalVector = std::array<double, 3>;
 
+    /// Vertical extent, horizontal extent, and width of initial fracture at
+    /// a seed point.
+    class SeedSize
+    {
+    private:
+        /// Named quantity indices.
+        enum class Quant : std::size_t {
+            /// Vertical extent.
+            Vertical,
+
+            /// Horizontal extent.
+            Horizontal,
+
+            /// Initial fracture width.
+            Width,
+
+            /// Number of named quantities.
+            ///
+            /// Note to maintainers: NumQuant *MUST* be last enumerator.
```

Comment:
Num

Reply:
> Num

:facepalm: 

Thanks!  I've pushed an update to address this.